configs/_base_/default_runtime.py

```diff
@@ -19,3 +19,8 @@
 opencv_num_threads = 0
 # set multi-process start method as `fork` to speed up the training
 mp_start_method = 'fork'
+
+# The default batch size of configs in MMDet.
+default_batch_size = 16
```

Comment:
The `default_batch_size` may be a little confusing. Although we have some comments to explain what it's for, users may considier it as the batch size setting but find the actual function is change the learning rate.
I think it's better to make it as a field of optimizer settings.

Reply:
agree.
We can add comments above `optimizer.lr` to indicate users that if they want to modify optimizer.lr, they may also modify default_batch_size.

Reply:
If this option is only used for auto-scaling, I prefer setting something like `default_num_gpu=8` in config. So that user will know this config, by default, is trained on 8 GPU, and auto-scaling will be based on `world_size/ default_num_gpu`. 
Further, `default_num_gpu`, `samples_per_gpu`,  `world_size` are completely decoupled. They are respectively defined by mmdet's official configuration (a constant to user), user's training preference and user's distributed environment. In contrast, default_batch_size is `default_num_gpu * samples_per_gpu`, which is coupled.

Reply:
> The `default_batch_size` may be a little confusing. Although we have some comments to explain what it's for, users may considier it as the batch size setting but find the actual function is change the learning rate. I think it's better to make it as a field of optimizer settings.

Done

Reply:
In my lastest commit, I made `default_num_gpu` as the specific setting of auto-scaling in `auto_scale_lr_config`. But reviews from both of you are really helpful to me that I decided to make all the config in one dict.